docs/behind_scenes/d3d12_support.rst

```diff
@@ -21,7 +21,7 @@ RenderDoc has initial support for D3D12, but it contains some caveats. In additi
 * RenderDoc captures may not be portable between different systems, only currently supporting capture and replay on the same or similar enough machines.
 * Reserved resources are not supported.
 * Pixel history is not implemented.
-* Shader debugging is not currently supported.
+* Shader debugging is supported only for shaders compiled with FXC (Shader Model 5.1 or lower).
```

Comment:
I think we can just drop this note entirely, since DXIL support is already non-existant as noted below. If/when DXIL support is added, we can add a caveat indicating that it's not debuggable.